src/ontology/mondo-edit.obo

```diff
@@ -366864,10 +366864,10 @@ xref: NCIT:C129739 {source="MONDO:equivalentTo"}
 xref: Orphanet:183625 {source="MONDO:equivalentTo"}
 xref: UMLS:C3888631 {source="MEDGEN:1392102", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 is_a: MONDO:0003847 {source="https://github.com/monarch-initiative/mondo/issues/1022"} ! hereditary disease
-intersection_of: MONDO:0005015 ! diabetes mellitus
-intersection_of: has_characteristic MONDO:0021152 ! inherited
 relationship: curated_content_resource https://search.clinicalgenome.org/kb/conditions/MONDO:0015967 {source="MONDO:CLINGEN"}
+relationship: has_characteristic MONDO:0021152 {source="https://orcid.org/0000-0001-5208-3432"} ! inherited
```

Comment:
@nicolevasilevsky. I think we still need to have "diabetes" as a parent. 
I will update it.